workflow-service/src/main/java/com/redhat/parodos/workflow/execution/dto/WorkFlowRequestDTO.java

```diff
@@ -47,6 +47,12 @@ public class WorkFlowRequestDTO {
 
 	private List<WorkRequestDTO> works;
 
+	/**
+	 * The ID of the flow that lead to execution of this workflow. Useful for passing
+	 * context from execution to execution
+	 */
+	private UUID invokingExecutionID;
```

Comment:
pls rename to invokingExecutionId (lowe d at the end to match convention `projectId`, `workflowExecutionId`...)